test-tools/test_state_recover.sh

```diff
@@ -177,38 +184,68 @@ function step2_delete_old_state()
     echo "Created signed state amount: $signed_state_amount"
 
     #random delete some states
-    random_num=$(( ( RANDOM % ($signed_state_amount - 1) + 1 ) ))
+    state_delete_num=$(( ( RANDOM % ($signed_state_amount - 1) + 1 ) ))
```

Comment:
If it happens to have only one round in data/saved/$PACKAGE/0/123, the RANDOM % ($signed_state_amount - 1) will cause a division by 0 error.

Reply:
It is impossible to happen with current stateSavePeriod (60 seconds) and test run time (10 > minutes)

Reply:
Got it.